lib/rust/ensogl/core/src/display/navigation/navigator.rs

```diff
@@ -29,25 +41,34 @@ pub struct NavigatorModel {
     /// Indicates whether events handled the navigator should be stopped from propagating further
     /// after being handled by the Navigator.
     disable_events:  Rc<Cell<bool>>,
+    max_zoom_limit:  Rc<Cell<Option<f32>>>,
```

Comment:
`max_zoom_limit` suggests that we have some `max_zoom` which may vary, but its variance is limited (hence limit). This is not how this parameter works. Please rename to just `max_zoom`.

Reply:
Fixed